fabfile.py

```diff
@@ -103,35 +94,26 @@ def help(name=""):
 
 @task
 def production(stackname=None):
-    if not stackname:
-        stackname = 'blue'
-
     """Select production environment"""
     env['environment'] = 'production'
     env['aws_migration'] = False
-    _set_gateway('publishing.service.gov.uk')
+    env.gateway = 'jumpbox.publishing.service.gov.uk'
 
 
 @task
 def staging(stackname=None):
-    if not stackname:
-        stackname = 'blue'
-
     """Select staging environment"""
     env['environment'] = 'staging'
     env['aws_migration'] = False
-    _set_gateway('staging.publishing.service.gov.uk')
+    env.gateway = 'jumpbox.staging.publishing.service.gov.uk'
 
 
 @task
 def integration(stackname=None):
-    if not stackname:
-        stackname = 'blue'
-
     """Select integration environment"""
     env['environment'] = 'integration'
     env['aws_migration'] = True
-    _set_gateway("{}.integration.govuk.digital".format(stackname))
+    env.gateway = 'jumpbox.blue.integration.govuk.digital'
```

Comment:
If we're no longer using the `stackname` parameter, I think we should remove it from the argument list to avoid any confusion.

Reply:
Oh yeah - thanks for spotting. I should have been paying more attention :-|.